### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ammaratef45**
Clicking the badge opens another repo, I've fixed that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 [![PRs Welcome][prs-badge]](http://makeapullrequest.com)
 
 [![Watch on GitHub](https://img.shields.io/github/watchers/zino-app/graphql-flutter.svg?style=flat&logo=github&colorB=deeppink&label=Watchers)](https://github.com/felangel/bloc)
```
Comment (HofmannZ) [inline README.md]: Same change should be done here

**Comment by ammaratef45** [comment]
@HofmannZ Is it fine now?

**Pull request merged**